android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/PatientDetailsActivity.kt

```diff
@@ -75,39 +81,31 @@ class PatientDetailsActivity : BaseMultiLanguageActivity() {
         AncDetailsViewModel(patientRepository, patientId = patientId).createFactory()
       )[AncDetailsViewModel::class.java]
 
+    activityAncDetailsBinding.patientDetailsToolbar.setNavigationOnClickListener { onBackPressed() }
+  }
+
+  override fun onResume() {
+    super.onResume()
     activityAncDetailsBinding.txtViewPatientId.text = patientId
 
+    loadProgress.show()
+
     ancDetailsViewModel
       .fetchDemographics()
       .observe(this@PatientDetailsActivity, this::handlePatientDemographics)
-
-    adapter =
-      ViewPagerAdapter(
-        supportFragmentManager,
-        lifecycle,
-        bundleOf(Pair(QuestionnaireActivity.QUESTIONNAIRE_ARG_PATIENT_KEY, patientId))
-      )
-    activityAncDetailsBinding.pager.adapter = adapter
-
-    TabLayoutMediator(activityAncDetailsBinding.tablayout, activityAncDetailsBinding.pager) {
-        tab,
-        position ->
-        tab.text = details[position]
-      }
-      .attach()
-
-    activityAncDetailsBinding.patientDetailsToolbar.setNavigationOnClickListener { onBackPressed() }
   }
 
   override fun onCreateOptionsMenu(menu: Menu?): Boolean {
-    menuInflater.inflate(R.menu.profile_menu_madx, menu)
+    menuInflater.inflate(R.menu.profile_menu, menu)
     return true
   }
 
   override fun onPrepareOptionsMenu(menu: Menu?): Boolean {
     val removeThisPerson = menu!!.findItem(R.id.remove_this_person)
-    val ancEnrollment = menu!!.findItem(R.id.mark_as_anc_client)
-    ancEnrollment.isVisible = !isPregnant
+    val ancEnrollment = menu!!.findItem(R.id.anc_enrollment)
+    val pregnancyOutcome = menu!!.findItem(R.id.pregnancy_outcome)
+    if (isMale) pregnancyOutcome.isVisible = false else ancEnrollment.isVisible = isPregnant
+    if (isMale) ancEnrollment.isVisible = false else ancEnrollment.isVisible = !isPregnant
```

Comment:
something looks off here, if `isMale` is false, the `ancEnrollment.isVisible` assigned on line 107 will always be overwritten by line 108, wouldn't the below give the equivalent with less code?
```suggestion
    if (isMale) pregnancyOutcome.isVisible = false
    ancEnrollment.isVisible = if (isMale) false else !isPregnant
```